### PR TITLE
ADAPT-2726: Remplace focal point field with smart/manual focal point

### PR DESCRIPTION
**Pull request by Lyafel**
# NOT READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to...
3. Verify...

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by Lyafel** [comment]
@yvonnetangsu I'm not sure about my local settings or what is it about but isNetlify in transformImage.js is always false for me so it always returns a.storyblok instead of img2.storyblok, I removed that line to test the changes but not sure if I need to do anything about it in the future

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu I'm not sure about my local settings or what is it about but isNetlify in transformImage.js is always false for me so it always returns a.storyblok instead of img2.storyblok, I removed that line to test the changes but not sure if I need to do anything about it in the future

Thanks, @Lyafel I'll pull down tomorrow to take a look. It is normal that you won't see the results in local, only in the Netlify preview. Please feel free to publish a page to get a Netlify preview link.

File: src/components/media/cardImage.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ const CardImage = ({
   loading,
   ...props
 }) => {
-  const imgFocus = objectPosition[imageFocus] ?? objectPosition.center;
```
Comment (sherakama) [inline src/components/media/cardImage.js:16]: @Lyafel, 

I think we will need to keep the object position functionality to ensure we don't break anything we currently have. Can you restore this functionality and add the smart/custom-focal point parameters to the request url.

Comment (sherakama) [reply src/components/media/cardImage.js:16]: @yvonnetangsu,

Keeping the object position and adding in the smart cropping should help maintain what we have plus add the value of the focal point cropping.

**Pull request closed without merging**